src/main/scala/dotty/renderers/ScalaHtmlRenderer.scala

```diff
@@ -29,8 +29,7 @@ class SignatureRenderer(pageContext: ContentPage, sourceSetRestriciton: JSet[Dis
     def renderLink(name: String, dri: DRI, modifiers: scalatags.Text.all.Modifier*) =
         link(dri) match
             case Some(link) => a(href := link, modifiers)(name)
-            case None if modifiers.isEmpty => raw(name)
-            case _ => span(modifiers)(name)
+            case _ => span(attr("data-unresolved-link") := dri.toString, modifiers)(name)
```

Comment:
Why do we need `data-unresolved-link` in such case? TBH I prefer if signature that does not link to anything looks like text not a link

@TheElectronWill maybe you can help us here with your UX wisdom?

Reply:
It's easier to manage style from custom CSS directives. To do so, I have to somehow determine which fragments are supposed to be signature, but their `DRI` could not be resolved. What's more, in dokka originally all signatures that are not resolved are marked that way, so I figured out it is convenient to add similar label here

Reply:
Unresolved links shouldn't look the same as resolved ones. The `data-unresolved-link` is fine to me (even more so if that's what dokka does everywhere), but the CSS styles associated with it should be changed.